frontend/device/remarkable/device.lua

```diff
@@ -301,7 +335,12 @@ function Remarkable:setEventHandlers(UIManager)
     end
 end
 
-if isRm2 then
+if isRmPaperPro then
+    if not os.getenv("LD_PRELOAD") then
```

Comment:
There's no other/better way to check?

Reply:
I can't think better way to check shim.so (or whatever the user names it) so I decided it was the best compromise for now.

Reply:
You should be able to pull the platform/board name (ferrari IIRC?) from *somewhere* in procfs (possibly somewhere near `/sys/firmware/devicetree/base/compatible`).